pkg/local_object_storage/shard/gc.go

```diff
@@ -117,16 +117,13 @@ func (gc *gc) listenEvents() {
 		v.cancelFunc()
```

Comment:
we do not need this func now then cause it can not work now. again, it is more like a disadvantage for me, cause i do not see any context restrictions here and the whole code looks like a context-based one, despite all the operations ignore contexts

Reply:
FS operations aren't context-bound. (local) DB operations aren't context-bound. Local object storage works with FS and DBs.